refinery/tool_manager/models.py

```diff
@@ -122,15 +125,12 @@ class ToolDefinition(models.Model):
     )
 
     uuid = UUIDField(unique=True, auto=True)
-    name = models.TextField(unique=True, max_length=100, blank=False,
-                            null=False)
-    description = models.TextField(unique=True, max_length=500, blank=False,
-                                   null=False)
-    tool_type = models.CharField(max_length=100, choices=TOOL_TYPES,
-                                 blank=False, null=False)
-    file_relationship = models.ForeignKey("FileRelationship")
-    output_files = models.ManyToManyField("OutputFile")
-    parameters = models.ManyToManyField("Parameter")
+    name = models.TextField(unique=True, max_length=100)
+    description = models.TextField(unique=True, max_length=500)
```

Comment:
OK but keep in mind that you will have to:
1. document this feature
1. write additional code to handle violations of this constraint